renovate.json5

```diff
@@ -74,6 +74,20 @@
       "groupSlug": "all-non-major-gte-1.0",
       "automerge": true,
     },
+    // We're currently constrained in our ability to update the `tracing`
+    // packages to the latest versions because of our usage.  This should put
+    // those pacakges into their own group and away from other dep updates.
+    {
+      "matchManagers": [
+        "cargo"
+      ],
+      "matchPackagePatterns": [
+        "^tracing"
```

Comment:
the `opentelemetry-*` should also be in there since they are closely linked to the `tracing-opentelemetry` version

Reply:
@Geal Is that inclusive of `opentelemetry`?  (Basically, should I do `^opentelemetry` or `^opentelemetry-`?)

Reply:
yes, `^opentelemetry`

Reply:
Done with https://github.com/apollographql/router/pull/1509/commits/857fe1683e63d60f8ea316ccf7ad4fe0f61c8c3d.  Thanks!